test/data/testBasis.py

```diff
@@ -46,30 +52,35 @@ def testsizeofbasis(self):
         """Is the basis set the correct size?"""
 
         total = 0
-        multiple = self.multiple
-        if self.spherical:
-            multiple = self.multiple_spher
-
+        multiple = self.multiple_spher if self.spherical else self.multiple
         for atom in self.data.gbasis:
-
-            for fns in atom:
-
-                 # Add 3 for P, 5 or 6 for D, and so forth.
-                ftype = fns[0]
+            for (ftype, contraction) in atom:
                 total += multiple[ftype]
 
         self.assertEquals(self.data.nbasis, total)
 
+    def testcontractions(self):

```

Comment:
I have to admit I'm confused right now, not because these tests cause non-DALTON failures, but because these tests only seem to be in this branch/PR and not in master. How did they get in here? I can't find them in the commit list.